src/plugins/intel_gpu/include/intel_gpu/op/fully_connected_compressed.hpp

```diff
@@ -18,12 +18,14 @@ class FullyConnectedCompressed : public FullyConnected {
 
     FullyConnectedCompressed(const ov::Output<Node> &A,
                              const ov::Output<Node> &B,
+                             const ov::Output<Node> &bias,
                              const ov::Output<Node> &decompression_scale,
                              const ov::Output<Node> &decompression_zero_point,
                              const ov::element::Type output_type = ov::element::undefined);
 
     FullyConnectedCompressed(const ov::Output<Node> &A,
                              const ov::Output<Node> &B,
+                             const ov::Output<Node> &bias,
                              const ov::Output<Node> &decompression_scale,
                              const ov::element::Type output_type = ov::element::undefined);
```

Comment:
Since `Placeholder` op was added recently, we may now have single c-tor for this op with all arguments. Could you modify that too? Can be done in a separate PR if you want